tests/lib/utils/context.php

```diff
@@ -47,6 +47,24 @@ public function test__is_maintenance_mode__yep() {
 		$this->assertTrue( $actual_result );
 	}
 
+	public function test__is_overdue_locked__nope() {
+		// Note: `VIP_OVERDUE_LOCKOUT` not defined
+
+		self::assertFalse( defined( 'VIP_OVERDUE_LOCKOUT' ) );
+
+		$actual_result = Context::is_overdue_locked();
+
+		$this->assertFalse( $actual_result, '`VIP_OVERDUE_LOCKOUT` constant should not be defined for this test' );
+	}
+
+	public function test__is_overdue_locked__yep() {
+		Constant_Mocker::define( 'VIP_OVERDUE_LOCKOUT', true );
+
+		$actual_result = Context::is_overdue_locked();
+
+		$this->assertTrue( $actual_result );
```

Comment:
If we want to keep with the theme of having a custom error message for the assertion, we can do this:

```suggestion
		$this->assertTrue( $actual_result, 'The `VIP_OVERDUE_LOCKOUT` constant is set to `true`, but site is not reported as locked' );
```